src/openrct2/drawing/Text.cpp

```diff
@@ -247,3 +247,28 @@ int32_t gfx_draw_string_centred_wrapped(
 
     return layout.GetHeight();
```

Comment:
Please make the non `ScreenCoordsXY` function for `gfx_draw_string_centred_wrapped` call the new one, so we don't duplicate code.